packages/extension-browser/src/devtools/utils/analytics.ts

```diff
@@ -1,26 +1,46 @@
-import { AppInsights } from 'applicationinsights-js';
+import { ApplicationInsights } from '@microsoft/applicationinsights-web-basic';
+
+const instrumentationKey = '8ef2b55b-2ce9-4c33-a09a-2c3ef605c97d';
+
+let appInsights: ApplicationInsights;
+
+const trackEvent = (name: string, properties?: { [key: string]: string }, measurements?: { [key: string]: number }) => {
+    if (!appInsights) {
+        return;
+    }
+
+    appInsights.track({
+        baseData: {
+            measurements,
+            name,
+            properties
+        },
+        baseType: 'EventData',
+        name: `Microsoft.ApplicationInsights.${instrumentationKey}.Event`
+    });
+};
 
 /** Called to initialize the underlying analytics library. */
 export const setup = () => {
-    AppInsights.downloadAndSetup!({ instrumentationKey: '8ef2b55b-2ce9-4c33-a09a-2c3ef605c97d' });
+    appInsights = new ApplicationInsights({ instrumentationKey });
 };
 
 /** Called when analysis was canceled by the user. */
 export const trackCancel = (duration: number) => {
-    AppInsights.trackEvent('f12-cancel', undefined, { 'f12-cancel-duration': duration });
+    trackEvent('f12-cancel', undefined, { 'f12-cancel-duration': duration });
 };
 
 /** Called when analysis finished. */
 export const trackFinish = (duration: number) => {
-    AppInsights.trackEvent('f12-finish', undefined, { 'f12-finish-duration': duration });
+    trackEvent('f12-finish', undefined, { 'f12-finish-duration': duration });
 };
 
 /** Called when the "Hints" tab was opened by the user. */
 export const trackShow = () => {
-    AppInsights.trackEvent('f12-show');
+    trackEvent('f12-show');
 };
 
 /** Called when analysis was started by the user. */
 export const trackStart = () => {
-    AppInsights.trackEvent('f12-start');
+    trackEvent('f12-start');
```

Comment:
Can we have an "install" event? Or something that gets send the first time the user goes to the `hint` pannel.

Reply:
We could do a "first-show" event, with the caveat that it would be reset by clearing data for or re-installing the extension. 

For true "installs" we should rely on the dashboards for whichever extension stores we're published in. I'd probably prefer going this route and not adding another event at the moment. Thoughts?

Reply:
I should add that application insights also tags unique "users" so we can probably get a rough count of installs that way as well without adding another event.

Reply:
OK, then let's not do anything else here :)